geometry/geometry_config.py

```diff
@@ -606,31 +606,58 @@
         c.tauMudet.NFethick = 4 #upstream slabs, more thick
         c.tauMudet.NFethin = 4 #downstream slabs, less thick
         c.tauMudet.NRpc= 8
-                
-        c.tauMudet.XFe = scaleMudet*1.900*u.m #layer dimensions, excluded supports
-        c.tauMudet.YFe = scaleMudet*3.600*u.m
+        
+        c.tauMudet.XFe = scaleMudet*1.950*u.m #layer dimensions, excluded supports
+        c.tauMudet.YFe = scaleMudet*3.850*u.m
+
         c.tauMudet.ZFethick = 15.*u.cm
         c.tauMudet.ZFethin = 10.* u.cm
 
         c.tauMudet.XRpc = c.tauMudet.XFe
         c.tauMudet.YRpc = c.tauMudet.YFe
-        c.tauMudet.ZRpc = 7.*u.cm
+        c.tauMudet.ZRpc = 8.*u.cm
         #support structure
-        c.tauMudet.UpperSupportX = 34 * u.cm
-        c.tauMudet.UpperSupportY = 34 * u.cm
-        c.tauMudet.LowerSupportX = 34 * u.cm
-        c.tauMudet.LowerSupportY = 34 * u.cm
-        c.tauMudet.LateralSupportX = 34 * u.cm
-        c.tauMudet.LateralSupportY = 34 * u.cm
+        c.tauMudet.UpperSupportX = 30 * u.cm
+        c.tauMudet.UpperSupportY = 32 * u.cm
+        c.tauMudet.LowerSupportX = 30 * u.cm
+        c.tauMudet.LowerSupportY = 40 * u.cm
+        c.tauMudet.LateralSupportX = 30.5 * u.cm
+        c.tauMudet.LateralSupportY = 32 * u.cm
 
         c.tauMudet.Xtot = c.tauMudet.XFe + 2 * c.tauMudet.LateralSupportX#now we need to include also supports.
         c.tauMudet.Ytot = c.tauMudet.YFe + c.tauMudet.UpperSupportY + c.tauMudet.LowerSupportY 
         c.tauMudet.Ztot = c.tauMudet.NRpc*c.tauMudet.ZRpc+c.tauMudet.NFethick*c.tauMudet.ZFethick + c.tauMudet.NFethin*c.tauMudet.ZFethin
         #c.tauMudet.zMudetC = -c.decayVolume.length/2. - c.tauMudet.Ztot/2
         c.tauMudet.zMudetC = c.Chamber1.z -c.chambers.Tub1length - c.tauMudet.Ztot/2 -31*u.cm;
         #lateral cuts
-        c.tauMudet.CutHeight = 100 * u.cm
-        c.tauMudet.CutLength = 25 * u.cm
+        c.tauMudet.CutHeight = 78.548 * u.cm
+        c.tauMudet.CutLength = (c.tauMudet.CutHeight / 2) / (r.TMath.Tan(r.TMath.DegToRad(55)))
```

Comment:
```suggestion
        c.tauMudet.CutLength = (c.tauMudet.CutHeight / 2) / (r.TMath.Tan(r.TMath.DegToRad() * 55))
```

Sorry, misremembered the [function signature](https://root.cern/doc/v616/namespaceTMath.html#a9056aad8f51639e46d4a7d2c16645928)